addons/l10n_it_edi/models/account_edi_format.py

```diff
@@ -460,16 +515,16 @@ def _import_fattura_pa(self, tree, invoice):
                             domains.append([('l10n_it_codice_fiscale', '=', self.env['res.partner']._l10n_it_normalize_codice_fiscale(codice))])
                         partner = elements and self.env['res.partner'].search(
                             AND([OR(domains), OR([[('company_id', '=', company.id)], [('company_id', '=', False)]])]), limit=1)
-                if not partner:
+                if incoming and not partner:
```

Comment:
But we still try to find a partner for the customer invoices?

Reply:
This is the section that looks for the **Sender**'s (`DatiTrasmissione`) email.
To me it doesn't even make sense. **Sender** can be **Aruba** I think.

Reply:
In both incoming and outgoing cases we look up the partner via the VAT and codice fiscale the same way.
In case we are the buyer we try 1 additional (pre-existing) method: We look at who transmitted the invoice (which is the seller and thus not useful to find a partner in case "we" are the seller.). (Previously we were always the buyer.)
As far as my understanding of this goes it does not hurt. Whether it is really useful or not I can't really say.

Should I remove it?

Reply:
It makes sense to keep it as it is consistent with what we had before.